core/loaders/module.ts

```diff
@@ -22,3 +33,19 @@ export default async function (path: string): Promise<Data> {
 
   return data;
 }
+
+function getDependencies(path: string, result: Deno.EmitResult) {
+  const { sources } = JSON.parse(result.files[`${bundle}.map`]) as SourceMap;
+  const filePath = `${fileProtocol}${path}`;
+
+  return sources
+    .filter((x) => x !== filePath && x.startsWith(fileProtocol))
+    .map((x) => x.substr(fileProtocol.length));
+}
+
+function executeModule(result: Deno.EmitResult) {
+  const code = result.files[bundle] as string;
+  const metadata = "data:application/javascript;charset=utf-8";
+  const encoded = `${metadata},${encodeURIComponent(code)}`;
+  return import(encoded);
```

Comment:
This is a good candidate to be moved to `core/helpers` so we can reuse it for other loaders.